flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/RowDataToProtoSerializer.java

```diff
@@ -135,7 +138,7 @@ public Object toProtoValue(
                                 element.getInt(fieldNumber), false);
                     } else {
                         return AvroToProtoSerializer.AvroSchemaHandler.convertTime(
-                                element.getLong(fieldNumber), true);
+                                ((long) element.getLong(fieldNumber)), true);
```

Comment:
What is otherwise returned by `getLong`?

Reply:
Removed the redundant cast. Thanks!